src/frontend/src/icp/services/ic-custom-tokens.services.ts

```diff
@@ -41,3 +46,53 @@ export const saveCustomTokens = async ({
 	// Reload all custom tokens for simplicity reason.
 	await loadUserTokens({ identity });
 };
+
+export const saveIcrcCustomToken = async ({
+	tokens,
+	updateSaveProgressStep,
+	modalNext,
+	onSuccess,
+	onError
+}: {
+	tokens: Pick<IcrcCustomToken, 'enabled' | 'version' | 'ledgerCanisterId' | 'indexCanisterId'>[];
+	updateSaveProgressStep: (step: ProgressStepsAddToken) => void;
+	modalNext: () => void;
+	onSuccess: () => void;
+	onError: () => void;
+}) => {
+	const $i18n = get(i18n);
+	const $authStore = get(authStore);
```

Comment:
can you pass the `identity` as parameter of the function for consistency reason?

Reply:
doing it, and do you want to keep the same name for the `updateSaveProgressStep` and call it `progress`?

Reply:
Ah yeah why not, it's generally what I used so far

Reply:
cool